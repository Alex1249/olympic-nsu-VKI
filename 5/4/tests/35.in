26
l w n b u y s c g a k j z o h f r v p d q t e i x m
