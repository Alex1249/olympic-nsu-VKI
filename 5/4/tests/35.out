l w n b u y s c g a k j z o h f r v p d q t e m i x
